odata-protocol/11.4 Data Modification.md

```diff
@@ -148,6 +148,21 @@ to that collection's URL. The `POST` body MUST contain a single valid
 representation of an entity of the declared target entity type,
 or one of its derived types.
 
+If the service is unable to determine the canonical collection (as defined in [section ##ContextURL])
+for the entity, it MUST fail the request.
+This determination is possible, for example, if the collection's URL
+- consists of an entity set
+- ends with a containment navigation property
+- ends with a non-containment navigation property with a navigation property binding
```

Comment:
Is the navigation property binding really sufficient? Given
```xml
<EntitySet Name="Orders">
  <NavigationPropertyBinding Path="DeepLink" Target="ns.container/Orders/Items/Subitems" />
</EntitySet>
```
how can a `POST Orders(23)/DeepLink` determine the `x` in
```
Location: http://server/Orders(23)/Items(x)/Subitems(y)
```
for the newly-created entity? (`y` is generated by the service.)